lib/ts/BufferWriter.h

```diff
@@ -652,6 +652,22 @@ bwformat(BufferWriter &w, BWFSpec const &, char c)
   return w.write(c);
 }
 
+inline BufferWriter &
+bwformat(BufferWriter &w, BWFSpec const &spec, bool f)
+{
+  static const ts::string_view txt_true{"true"};
+  static const ts::string_view txt_false{"false"};
+  static const ts::string_view txt_TRUE{"TRUE"};
+  static const ts::string_view txt_FALSE{"FALSE"};
+  if ('s' == spec._type)
+    w.write(f ? txt_true : txt_false);
+  else if ('S' == spec._type)
+    w.write(f ? txt_TRUE : txt_FALSE);
+  else
+    bw_fmt::Format_Integer(w, spec, static_cast<uintmax_t>(f), false);
+  return w;
```

Comment:
why is there no alignment done here? `Format_Integer` writes with alignment.  either alignment should be done here, or the `spec` should be adjusted so that it is not alignment is not done twice.

Reply:
^ this seems to be handled in `Do_Alignment`. ok.

Reply:
Yes, in general `Do_Alignment` handles alignment (which is why I recommended not doing alignment for unnormal floating point values). Doing alignment explicitly can be more efficient and is used for numeric types where performance is critical.